modules/ossm-rn-new-features.adoc

```diff
@@ -19,11 +19,101 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 2.2
+
+This release of {SMProductName} adds new features and enhancements, and is supported on OpenShift Container Platform 4.9, and 4.10.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
+
+|===
+|Component |Version
+
+|Istio
+|1.12.7
+
+|Envoy Proxy
+|1.20.4
+
+|Jaeger
+|1.30.2
+
+|Kiali
+|1.48.0.16
+|===
+
+=== `WasmPlugin` API
+This release changes the implementation of `WebAssembly` extensions to use the `WasmPlugin` API instead of the `ServiceMeshExtention` API, which has been deprecated.
+
+=== ROSA support
+This release introduces service mesh support for Red Hat OpenShift on AWS (ROSA), including multi-cluster federation.
+
+=== `istio-node` DaemonSet renamed
+This release, the `istio-node` DaemonSet is renamed to `istio-cni-node` to match the name in upstream Istio.
+
+=== Envoy sidecar networking changes
+Istio 1.10 updated Envoy to send traffic to the application container using `eth0` rather than `lo` by default.
+
+=== Service Mesh Control Plane 1.1
+This release marks the end of support for service mesh control planes based on Service Mesh 1.1 for all platforms.
+
+=== Istio 1.12 Support
+
+{SMProductShortName} 2.2 is based on Istio 1.12, which brings in new features and product enhancements. While many Istio 1.12 features are supported, the following exceptions should be noted:
+
+* AuthPolicy Dry Run is a tech preview feature.
+* gRPC Proxyless Service Mesh is a tech preview feature.
+* Telemetry API is a tech preview feature.
+* Discovery selectors is not a supported feature.
+* External control plane is not a supported feature.
+* Gateway injection is not a supported feature.
+
+=== Kubernetes Gateway API
```

Comment:
Why is this configuration listed here and not in the Technology Preview module?

Reply:
@JStickler for this particular item, @longmuir and @dgn thought it would be a good idea to list it separate from the other items in the bulleted list. I thought it made the most sense to list it here, right after the other Istio TP items.

Reply:
Because it's part of this release? 

I didn't see the Technology Preview section - it looks like it is very out of date and needs a rethink. I think that should come after this release though.

Reply:
Maybe list as a bullet in the section above, and move the configuration information in the Technology Preview module.

Reply:
I think it's better to keep them together. We could even delete the "Technology Preview" section, as it is so out of date as to not be relevant.

Reply:
There's also precedent, as we've included instructions on other features in the release notes before. I think it's less than ideal, but this is just following the pattern we started with other releases.

Reply:
@JStickler why would we separate that information? I think it would be harder for customers to find the configuring information that way. I think it's best to keep the information with the API Gateway topic.